src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -3895,6 +3899,80 @@ citus_internal_update_none_dist_table_metadata(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * citus_internal_database_command is an internal UDF to
+ * create/drop a database in an idempotent maner without
+ * transaction block restrictions.
+ */
+Datum
+citus_internal_database_command(PG_FUNCTION_ARGS)
+{
+	CheckCitusVersion(ERROR);
+	if (!ShouldSkipMetadataChecks())
+	{
+		EnsureCoordinatorInitiatedOperation();
+	}
+	PG_ENSURE_ARGNOTNULL(0, "database command");
```

Comment:
An easier way to disallow `NULL` is by marking the function as `STRICT` in it SQL definition. We usually only use `PG_ENSURE_ARGNOTNULL` if one of the arguments actually can be `NULL`.

Reply:
I want to discuss the STRICT parameter vs PG_ENSURE_ARGNOTNULL(0, "database command"); in citus_internal_database_command
At the latest comment @JelteF  wanted me to use STRICT. However, STRICT silently returns null and seems a successful execution since we don't check the output. However, PG_ENSURE_ARGNOTNULL(0, "database command"); returns error for NULL parameter
I think using PG_ENSURE_ARGNOTNULL makes more sense. What do you think @JelteF @onurctirtir ?

Reply:
Like I said in chat I'm fine with keeping PG_ENSURE_ARGNOTNULL the error should report the actual argument name though.
```suggestion
	PG_ENSURE_ARGNOTNULL(0, "command");
```